ext/reflection/php_reflection.c

```diff
@@ -1664,6 +1664,59 @@ ZEND_METHOD(ReflectionFunctionAbstract, getClosureScopeClass)
 }
 /* }}} */
 
+/* {{{ Returns an associative array containing the closures lexical scope variables */
+ZEND_METHOD(ReflectionFunctionAbstract, getClosureUsedVariables)
+{
+	reflection_object *intern;
+	const zend_function *closure_func;
+
+	if (zend_parse_parameters_none() == FAILURE) {
+		RETURN_THROWS();
+	}
+	GET_REFLECTION_OBJECT();
+
+	array_init(return_value);
+	if (!Z_ISUNDEF(intern->obj)) {
```

Comment:
Does intern->obj only get set for closures (and not, say, instance methods)?

Reply:
Okay, I see the same check elsewhere, so the answer must be "yes"...

Reply:
I think it does, this is based on same code as in other getClosure* methods ...